libnavigation-core/api/current.txt

```diff
@@ -539,7 +539,8 @@ package com.mapbox.navigation.core.trip.model.eh {
 
   public final class Edge {
     method public boolean getBridge();
-    method public String? getCountryCode();
+    method public String? getCountryCodeIso2();
```

Comment:
This is a major API-breaking change.

We should leave `getCountryCode`, but wire it to `getCountryCodeIso2` and deprecate.

Reply:
or wire to `getCountryCodeIso3`, not sure which one was returned earlier

Reply:
I'd say it was `countryCodeIso3` https://github.com/mapbox/mapbox-navigation-android/pull/4182#discussion_r602231457

Could you confirm @mskurydin @SiarheiFedartsou @korshaknn?

Reply:
`getCountryCodeIso3` is actually `countryCode`, and new one is `countryCodeIso2`

cc @mskurydin